anthos-onprem-terraform/avmw_user_cluster_metallb/terraform.tfvars.sample

```diff
@@ -2,12 +2,13 @@ project_id                  = "FLEET_HOST_PROJECT_ID"
 region                      = "REGION"
 admin_cluster_name          = "YOUR_ADMIN_CLUSTER_NAME"
```

Comment:
Change to ADMIN_CLUSTER_NAME so that it matches the placeholder used in the gcloud tab in the public doc: https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/create-user-cluster-api#create_a_user_cluster

Reply:
Updated. Thanks for the guidance!